components/switch/switch-visibility.js

```diff
@@ -0,0 +1,22 @@
+import '../icons/icon.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { SwitchMixin } from './switch-mixin.js';
+
+class VisibilitySwitch extends SwitchMixin(LitElement) {
+
+	constructor() {
+		super();
+		this.text = 'Visibility';
```

Comment:
If you want to languify this out of the gate, you could likely reuse the translations from [this term](https://search.d2l.dev/xref/lms/ipsct/D2L.IM.IPSCT/LangTerms/definitions/OnlineRooms.xml?r=d4b17a73#544).

Reply:
Thanks.  I think there might still be some conversation with Design.  In several places they've been flipping the text along with the state of the switch, which ends up being very confusing, and is not recommended. So... I plan to localize this sometime in the next couple of days.